README.md

```diff
@@ -1,3 +1,7 @@
+# This is a fork of https://github.com/latitudegames/GPT-3-Encoder. I made this fork so I could apply some PRs that had been sent to the upstream repo.
```

Comment:
Another spot where we could revert this change for cleaning it up to merge

Reply:
Yep, we don't want to pull in the changes from my fork.